src/test/rpc/GatewayBalances_test.cpp

```diff
@@ -28,123 +29,159 @@ class GatewayBalances_test : public beast::unit_test::suite
 {
 public:
     void
-    testGWB(FeatureBitset features)
+    testGWB(FeatureBitset features, unsigned int apiVersion)
```

Comment:
This really looks to me like two different tests. Consider keeping the original `testGWB()` as is and adding say `testGWBApiV2(FeatureBitset)`, which has the added code.